src/scribe_data/cli/cli_utils.py

```diff
@@ -52,6 +62,129 @@
 }
 
 
+def get_available_languages() -> list[tuple[str, str]]:
+    """
+    Get available languages from the data extraction folder.
+
+    Returns:
+        list[tuple[str, str]]: A list of tuples with the language name and its QID.
+    """
+    extraction_dir = LANGUAGE_DATA_EXTRACTION_DIR
+    available_languages = []
+    for lang_folder in extraction_dir.iterdir():
+        if lang_folder.is_dir():  # Check if it's a directory
+            lang_name = lang_folder.name
+            sparql_file_path = lang_folder / "verbs" / "query_verbs.sparql"
+            qid = extract_qid_from_sparql(sparql_file_path)
+            if qid:
+                available_languages.append((lang_name, qid))
+            else:
+                available_languages.append((lang_name, ""))
+    return available_languages
+
+
+def extract_qid_from_sparql(file_path: Path) -> str:
```

Comment:
`extract_qid_from_sparql`, `check_and_update_languages` and `update_language_metadata` wouldn't be needed as this would be covered by the check workflow on PRs that will assure that the metadata file is appropriate :) I'm still on the fence when it comes to `set_metadata` though 🤔

Reply:
Yeah, I agree with why we should move them. Great point 😄

I thought that the `set_metadata` would be useful if, for some reason, we could not automatically update the metadata(that is, when it was still in use), then we can call `scribe-data set-metadata -lang French -qid Q150` for example, and this updates our metadata file.